### PR TITLE
Add information to message in #reports

### PR DESCRIPTION
**Pull request by maloryware**
Adds date and changes formatting slightly.  Also corrects a minor spelling mistake (death sentence)

### CODE REVIEW

File: calSprite.py [hunk 1]
```diff
@@ -304,7 +312,9 @@ async def notice(self, text):
         msg = text[text.find(parameters[1][1:]) :]
 
         if msg.upper().startswith("REPORT"):
-            await self.send(f"PRIVMSG #reports {nick}: {msg}")
+            currTime = datetime.datetime.now
+            await self.send(f"PRIVMSG #reports {nick}: {msg} - at ",  )
```
Comment (maloryware) [inline calSprite.py]: the follow up commit fixes the fact that i completely forgot to finish this part

File: calSprite.py [hunk 2]
```diff
@@ -312,12 +312,18 @@ async def notice(self, text):
...
-            await self.send(f"PRIVMSG #reports {nick}: {msg} - at ",  )
-            
-            await self.send(f"PRIVMSG {nick} Report send :3 (mods can only see reports"
-                            " if they're online, might be better to ping "
-                            "a mod on discord if it's urgent)")
+            # tested these two lines below on IDLE, seemed to work well
+            inboundReport = msg.split(' ', 2)
+            reportMsgUser, reportMsgBody = inboundReport[1:]
+
+            # NOTE: untested since i didn't have the time to setup a local irc server (rip scratchware...)
```
Comment (maloryware) [inline calSprite.py]: shou please remind me to set up an unrealircd instance locally next time i talk about proposing changes to calsprite

**Comment by Dpeta** [comment]
Getting reportMsgUser/reportMsgBody this way unfortunately raises a ValueError if the report message is too short. Not every report will have a name & message.
```python
Task exception was never retrieved
future: <Task finished name='Task-18' coro=<CalSpriteBot.privmsg() done, defined at calSprite.py:204> exception=ValueError('not enough values to unpack (expected 2, got 1)')>
Traceback (most recent call last):
  File "calSprite.py", line 234, in privmsg
    reportMsgUser, reportMsgBody = inboundReport[1:]
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: not enough values to unpack (expected 2, got 1)
```
Privmsg reports that are done manually via the command in dms also don't enforce any syntax, so there's not going to be a handle as the first parameter.

**Comment by maloryware** [comment]
That should address everything

**Review by Dpeta — APPROVED** [review]
woo yeah woo yeah

**Pull request merged**